frontend/public/components/mounted-vol.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import {
+  getVolumeType,
+  getVolumeLocation,
+  getVolumeMountPermissions,
+  getVolumeMountsByPermissions,
+  getVolumeMountPath,
+  getVolumeSubPath,
+} from '../module/k8s/pods';
+import {
+  VolumeIcon,
+} from './utils';
+
+export const MountedVol: React.SFC<MountedVolsProps> = ({resourceVol, pod}) => {
+
+  const Volume = ({volume}) => {
+    const kind = _.get(getVolumeType(volume.volume), 'id', '');
+    const loc = getVolumeLocation(volume.volume);
+    const name = volume.name;
+    const mountPermissions = getVolumeMountPermissions(volume);
+
+    return <React.Fragment>
+      { pod ?
+        <div className="row">
+          <div className="col-sm-2 col-xs-4">{name}</div>
+          {volume.mounts.map((m, i) => <React.Fragment key={i}>
+            <div className="col-sm-3 hidden-xs">{getVolumeMountPath(m)}</div>
+            <div className="col-sm-2 hidden-xs">{getVolumeSubPath(m)}</div>
+          </React.Fragment>)}
+          <div className="col-sm-3 col-xs-4">
+            <VolumeIcon kind={kind} />
+            <span className="co-break-word">{loc && ` (${loc})`}</span>
+          </div>
+          <div className="col-sm-2 col-xs-4">{mountPermissions}</div>
+        </div>
+        :
+        <div className="row">
+          <div className="col-sm-3 col-xs-4">{name}</div>
+          <div className="col-sm-3 col-xs-4">
+            <VolumeIcon kind={kind} />
+            <span className="co-break-word">{loc && ` (${loc})`}</span>
+          </div>
+          <div className="col-sm-3 hidden-xs">{mountPermissions}</div>
+          <div className="col-sm-3 col-xs-4">
+            {volume.mounts.map((m, i) => <React.Fragment key={i}>
+              <div>{m.container}</div>
+            </React.Fragment>)}
+          </div>
+        </div>}
+    </React.Fragment>;
+  };
+
+  return <React.Fragment>
+    {_.size(getVolumeMountsByPermissions(resourceVol)) ?
+      <div className="co-m-table-grid co-m-table-grid--bordered">
+        <div className="row co-m-table-grid__head">
+          {pod ?
+            <div>
+              <div className="col-sm-2 col-xs-3">Name</div>
+              <div className="col-sm-3 col-xs-3">Mount Path</div>
+              <div className="col-sm-2 col-xs-3">SubPath</div>
+              <div className="col-sm-3 col-xs-3">Type</div>
+              <div className="col-sm-2 hidden-xs">Permissions</div>
+            </div>
+            :
+            <div>
+              <div className="col-sm-3 col-xs-3">Name</div>
+              <div className="col-sm-3 col-xs-3">Type</div>
+              <div className="col-sm-3 hidden-3">Permissions</div>
+              <div className="col-sm-3 col-xs-3">Utilized By</div>
+            </div>
```

Comment:
I'm not sure why these tables are different. We should be able to show the same things regardless of whether this is a pod. (For non-pod workloads, we simply don't make the container a link until we have a page to link to.)

Reply:
OK, I see. I'd still make them the same, which keeps the UI consistent and the code simpler.